### PR TITLE
Delete same notification if enter the channel.

### PR DESCRIPTION
**Pull request by SimYunSup**
## Specify project
client - RN.

## Description

Delete same notification if enter the channel.
And if user on the channel, delete the related notification.

https://user-images.githubusercontent.com/47362439/131440243-bfcf2a06-6c9a-4a9a-bd63-80433d818231.mp4

## Related Issues

issue #463

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/hackatalk/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn lint && yarn tsc`
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] I am willing to follow-up on review comments in a timely manner.

[WeHack] - [we go]

### CODE REVIEW

**Comment by hyochan** [comment]
@JongtaekChoi @geoseong Could you kindly review this work?

File: client/src/components/pages/Message.tsx [hunk 1]
```diff
@@ -488,6 +490,48 @@ const MessageScreen: FC = () => {
     params: {channelId},
   } = useRoute<RouteProp<MainStackParamList, 'Message'>>();
 
+  // Delete notification if user is foreground in channel.
+  useEffect(() => {
+    const subscription = Notifications.addNotificationReceivedListener(
+      (notification) => {
+        const parsedNotificationData = JSON.parse(
+          notification.request.content.data.data as string,
+        );
```
Comment (geoseong) [inline client/src/components/pages/Message.tsx:499]: compared to other similar codes like `L517-520` in this file and [L131-L134 in MainStackNavigator](https://github.com/dooboolab/hackatalk/blob/master/client/src/components/navigations/MainStackNavigator/index.tsx#L131-L134), this has only different style. do you have any special reason?

I think this kind of code style has to become the same.

**Review by JongtaekChoi — APPROVED** [review]
This change looks great. However, CodeCov bots warn you because no test code has been added. How about adding a few simple tests to this pool request?

File: client/src/components/pages/Message.tsx (hunk 1 above)
Comment (SimYunSup) [reply client/src/components/pages/Message.tsx:499]: I changed like you commented, but I think it went back to original. I'll fix it.

Comment (geoseong) [reply client/src/components/pages/Message.tsx:499]: > I changed like you commented, but I think it went back to original. I'll fix it.

I checked your commit, but no change was detected. please check if you'd pushed the commit to your repo.

and your other code in the same file [L517-L529](https://github.com/SimYunSup/hackatalk/blob/feature/delete_same_notification/client/src/components/pages/Message.tsx#L517-L529) looks similar to this code block. so could you tell me why do you think it went back to original? and also I don't know what is the original 😆.

please understand me if I ask the dumb question 🙏

**Review by hyochan — APPROVED** [review]
LGTM! Sorry for the delay 🙏

**Comment by hyochan** [comment]
@all-contributors Please add @SimYunSup for code.

**Pull request merged**